lib/index.js

```diff
@@ -4,6 +4,9 @@ import { WasmBoyController } from './controller/controller';
 import { WasmBoyMemory } from './memory/memory';
 import { saveCurrentAudioBufferToWav } from './debug/debug';
 
+// Get our package.json
+import packageJson from '../package.json';
```

Comment:
Actually can we make this a require? Travis gives an error from the import, since I don't think I have json imports set up in Babel: https://travis-ci.org/torch2424/wasmBoy/builds/444955301?utm_source=github_status&utm_medium=notification

So It would be `const packageJson = require('../package.json')`.

This is more my bad haha! 🙃

Reply:
Also, to test this, you could add a `console.log()` in `test/accuract/accuracy-test.js` near line 20, that simple logs out your new function 😄